src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGeneratorOptions.cs

```diff
@@ -61,12 +65,20 @@ public SwaggerGeneratorOptions()
 
         public IList<IParameterFilter> ParameterFilters { get; set; }
 
+        public IList<IParameterAsyncFilter> ParameterAsyncFilters { get; set; }
+
         public List<IRequestBodyFilter> RequestBodyFilters { get; set; }
 
+        public List<IRequestBodyAsyncFilter> RequestBodyAsyncFilters { get; set; }
+
         public List<IOperationFilter> OperationFilters { get; set; }
```

Comment:
```suggestion
        public IList<IOperationFilter> OperationFilters { get; set; }
```

Reply:
I have changed this aswell, however this is an existing property, isn't that technically an API break?

Reply:
I must have commented on the wrong line - the new properties should use `IList<T>` but anything already existing should be left alone.